### ISSUE TITLE
Error with angular 8 

### ISSUE THREAD

**Issue by StillUpgrade** [angular-material-extensions/password-strength#187]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?

> MacOs Mojave

### Versions

Angular CLI: 8.0.3
Node: 10.9.0
OS: darwin x64
Angular: 8.0.2
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
"@angular-material-extensions/password-strength": "^4.0.0-beta",

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect                0.800.3
@angular-devkit/build-angular        0.800.3
@angular-devkit/build-optimizer     0.800.3
@angular-devkit/build-webpack     0.800.3
@angular-devkit/core                        8.0.3
@angular-devkit/schematics        8.0.3
@angular/cdk                      8.0.1
@angular/cli                      8.0.3
@angular/material                 8.0.1
@ngtools/webpack                  8.0.3
@schematics/angular               8.0.3
@schematics/update                0.800.3
rxjs                              6.5.2
typescript                        3.4.5
webpack                           4.30.0

### Repro steps

Launch my app with ng build --watch, with or without aot, at the launch i have an error. Your component is not even used in the app.component.html but further in the signup page.

### The log given by the failure

![image](https://user-images.githubusercontent.com/10169092/59920551-295d5d80-942b-11e9-81fc-c2a40baacfad.png)

### Mention any other details that might be useful
If I can help you in anyway, ask for it. Thanks for your work.

**Comment by AnthonyNahas** [comment]
was the app working with the previous version ?

**Comment by StillUpgrade** [comment]
Yes

Le ven. 21 juin 2019 à 21:50, Anthony Nahas <notifications@github.com> a
écrit :

**Comment by AnthonyNahas** [comment]
Did u try to remove your node modules and install them again ?

**Comment by AnthonyNahas** [comment]
@StillUpgrade did u get the the error msg in your own app ? or u tried to serve the demo app locally ? I've just tested and patched the library to [v4.0.1](https://www.npmjs.com/package/@angular-material-extensions/password-strength)

If you are still facing the issue please consider to reopen this ticket #187

thank you
